File: JavaScript/rpc.js
const config = require('./config.json');
const RPC = require('discord-rpc');
const rpc = new RPC.Client({
    transport: 'ipc'
});

rpc.on('ready', () => {
    rpc.setActivity({
        details: "Hello there, stalker",
        state: "While you're here, check out my GitHub ",
        startTimestamp: new Date(), // Shows the elapsed time
        largeImageKey: "typing", 
        largeImageText: "Text for large image", // Text that shows when you hover over the large image
        smallImageKey: "Small image", 
        smallImageText: "Text for small image", // Text that shows when you hover over the small image
        buttons: [{
            label: "My GitHub", // Title of button 1
            url: "https://github.com/sydyen" // Link of button 1
        }, {
            label: "Button 2", // Title of button 2
            url: "https://youtube.com" // Link of button 2
        }]

    });
    console.log("RPC is running!") //Logs this message when the program is runnuing successfully
})

rpc.login(
    {
        clientId: config.ID //Your application id
    }
)

File: Python/rpc.py
from pypresence import Presence
import config
import time


clientID = config.ClientID # Your application id
RPC = Presence(clientID())
RPC.connect()
print('RPC is now running!') # Prints this message when the rpc is runnuing successfully

while True:
    RPC.update(large_image="Large Image", # 
                large_text="Large Image Text", # Text that shows when you hover over the large image
                small_image="Small Image", # 
                small_text="Small Image Text", # Text that shows when you hover over the large image
                details="Hello there, stalker!", 
                state="While you're here, check out my GitHub", 
                start=int(time.time()), # Shows the elapsed time
                buttons=[{
                    "label":"Button 1", # Title of button 1
                     "url":"https://github.com/sydyen" # Link of button 1
                     }, {
                    "label":"Button 2", # Title of button 2
                    "url":"https://github.com/sydyen" # Link of button 2
    }])
